libs/sdm-launchpad/assets/defaults.ini

```diff
@@ -0,0 +1,1241 @@
+##################### Grafana Configuration Defaults #####################
```

Comment:
maybe move this file to a grafana specific folder. Currently it looks like it's used by launchpad directly